lib/qa/authorities/linked_data/config.rb

```diff
@@ -63,6 +72,21 @@ def self.predicate_uri(config, key)
         pred_uri = RDF::URI(pred) unless pred.nil? || pred.length <= 0
         pred_uri
       end
+
+      private
+
+        def convert_1_0_to_2_0
+          convert_1_0_url_to_2_0_url(:search)
+          convert_1_0_url_to_2_0_url(:term)
+        end
+
+        def convert_1_0_url_to_2_0_url(action_key)
+          url_template = @authority_config.fetch(action_key, {}).fetch(:url, {}).fetch(:template, "")
+          return if url_template.blank?
+          warn "[DEPRECATED] #Linked data configuration #{authority_name} has 1.0 version format which is deprecated; update to version 2.0 configuration." \
+           "Called from #{Gem.location_of_caller.join(':')}."
+          @authority_config[action_key][:url][:template] = url_template.gsub("{?", "{")
```

Comment:
Does the conversion from 1.0 config to 2.0 config.  Adds a deprecation notice that the authority config needs to be updated.